src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.styles.ts

```diff
@@ -26,7 +26,8 @@ export const getStyles = (
             classNames.wrapper,
             {
                 flexGrow: 1,
-                position: 'relative'
+                position: 'relative',
+                padding: '16px 0px'
```

Comment:
What would you think of putting the spacing in a constant so we can update all the places in one go?